src/domain/hooks/helpers/event-cache.helper.ts

```diff
@@ -451,6 +489,28 @@ export class EventCacheHelper {
   ): Array<Promise<void>> {
     return [this.safeRepository.clearIsSafe(event)];
   }
+  /**
+   * Logs the number of unsupported chain events for each chain and clears the store.
+   */
+  private async _logUnsupportedEvents(): Promise<void> {
+    const chains = await this.chainsRepository.getChains();
+    await Promise.all(
+      chains.results.map(async (chain) => {
+        const cacheKey = CacheRouter.getUnsupportedChainEventCacheKey(
+          chain.chainId,
+        );
+        const count = await this.cacheService.getCounter(cacheKey);
+        if (count) {
```

Comment:
Yes, it should always be different from `0` (it should be a non-zero positive integer or `null`). But in case it's `0` (because the underlying implementation changes) then the `if` part shouldn't be executed, right? I mean, if it's `0` then we don't want to emit the log line. Wdyt? In that case the implementation would be accurate.

Reply:
Very true. I got confused thinking we were incrementing the counter. Let's leave it as is.